### ISSUE TITLE
document docker example for installing/configuring plugins

### ISSUE THREAD

**Issue by casz** [jfrog/artifactory-docker-examples#180]
Currently trying to configure buildSync.json and I have no idea what is the proper way of configuring the preinstalled buildSync plugin.

Atm I am getting replication errors for artifactory-build-info and I would like to configure it properly.

**Comment by casz** [comment]
reading the contents on the entrypoint-artifactory.sh reveals the secret but still not very clear.

```bash
addPlugins () {
    logger "Adding plugins if exist"
    cp -fv /tmp/plugins/* ${ARTIFACTORY_HOME}/etc/plugins/
}
```

**Comment by asinvrer** [comment]
?

Dragon

**Comment by eldada** [comment]
Yes. The secret is preparing your plugins on the host and mounting them to `/tmp/plugins/` in the Artifactory container. The plugins are copied to their final destination in the entrypoint script.
Indeed - this is not documented well here.
